src/content/docs/infrastructure/host-integrations/host-integrations-list/ansible-controller-integration.mdx

```diff
@@ -0,0 +1,184 @@
+
+      targets:
+        - description: Metrics of Ansible Automation Platform can be seen on the below url
+          urls: ["https://YOUR_HOST_IP:443/api/v2/metrics/?format=txt"]
+          use_bearer: true
+      #    tls_config:
+      #      ca_file_path: "/etc/etcd/etcd-client-ca.crt"
+      #      cert_file_path: "/etc/etcd/etcd-client.crt"
+      #      key_file_path: "/etc/etcd/etcd-client.key"
+
+      # Whether the integration should run in verbose mode or not. Defaults to false.
+      verbose: false
+
+      # Whether the integration should run in audit mode or not. Defaults to false.
+      # Audit mode logs the uncompressed data sent to New Relic. Use this to log all data sent.
+      # It does not include verbose mode. This can lead to a high log volume, use with care.
+      audit: true
+
+      # The HTTP client timeout when fetching data from endpoints. Defaults to 30s.
+      # scrape_timeout: "30s"
+
+      # Length in time to distribute the scraping from the endpoints.
+      scrape_duration: "5s"
+
+      # Number of worker threads used for scraping targets.
+      # For large clusters with many (&gt;400) endpoints, slowly increase until scrape
+      # time falls between the desired `scrape_duration`.
+      # Increasing this value too much will result in huge memory consumption if too
+      # many metrics are being scraped.
+      # Default: 4
+      # worker_threads: 4
+
+      # Whether the integration should skip TLS verification or not. Defaults to false.
+      insecure_skip_verify: true
+
+    timeout: 10s
+
+``` 
+
+</Collapser>
+
+<Collapser
+    id="step3"
+    title="Step 3. Forward Ansible controller logs to New Relic`"
+    >
+
+You can use our [log forwarding](/docs/logs/forward-logs/forward-your-logs-using-infrastructure-agent) to forward Ansible Automation Platform logs to New Relic. 
+
+Your log file is present in:
+/etc/newrelic-infra/logging.d/
+
+If you don't see your log file in the path listed above, then you will need to create a log file. 
+
+To create the log file, add the script below to the logging.yml file:
+
+```
+  - name: ansible-tower.log
+    file: /var/log/tower/tower.log
+    attributes:
+      logtype: ansible_tower_log
+
+```
+</Collapser>
+
+ <Collapser
+    id="step4"
+    title="Step 4. Restart your infrastructure agent"
+    >
+
+Before you can start reading your data use the instructions in our [infrastructure agent docs](/docs/infrastructure/install-infrastructure-agent/manage-your-agent/start-stop-restart-infrastructure-agent/) to restart your infrastructure agent. 
+</Collapser>
+
+<Collapser
+    id="step5"
+    title="Step 5. Use NRQL to view your data"
+    >
+You can use this NRQL query to view your Ansible Automation Platform data:
+
+```
+FROM Metric SELECT latest(awx_pending_jobs_total) AS 'Pending Jobs', latest(awx_running_jobs_total) AS 'Running Jobs'
+```
+
+</Collapser>
+</CollapserGroup>
+
+
+## See Ansible Automation Controller metrics in a dashboard [#dash]
```

Comment:
@homelessbirds you might set the context here with more information. It seems like if you install the infrastructure agent, configure the Prometheus OpenMetrics integration, and do some Ansible configurations, you still don't get the magical dashboards? What do you get out of the box? Do you then need to run the "Add data" I/O to get the dashboards, too?